spec/rack_test_spec.rb

```diff
@@ -145,6 +145,13 @@ module TestSessions
       end
     end
 
+    describe '#send_keys' do
+      it 'is a no-operation' do
+        session.visit('/form')
+        session.send_keys(:tab)
```

Comment:
Since it's a no-op, I'm not sure what the assertion should be. Would an `expect { }.not_to raise_error` be a good fit, or would we be better off skipping it like other JS-enabled tests?

Reply:
This should raise an error - calling it in the rack test driver makes no sense and would just lead to confusion